index.html

```diff
@@ -143,13 +143,15 @@ <h5>Priority L train subway Model</h5>
                         </a>
                 </div>
                 <div class="homebackground">
+                  <div class="container">
                 <h1>The Priority Fold</h1>
                 <h2 class="indenttext">A Priority Bike Product</h2>
-                <p class="indenttext3">Our Priority Fold has a lightweight, stripped down, simple but robust design, like our previous products, lowering testing costs. The bike can fold easily, allowing customers to save space at home in their apartment or when taking a crowded train. The accessories will be sold separately, allowing a cheap initial purchase and later customization when the customer feels comfortable with it.</p>
+                <p class="">Our Priority Fold has a lightweight, stripped down, simple but robust design, like our previous products, lowering testing costs. The bike can fold easily, allowing customers to save space at home in their apartment or when taking a crowded train. The accessories will be sold separately, allowing a cheap initial purchase and later customization when the customer feels comfortable with it.</p>
```

Comment:
it's personal preference, but I don't like .indentext3 here... though I am not a designer ;) @bingmo92